images/build/debian-iptables/distroless/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG GORUNNERIMAGE
+# Using directly debian base, as we are going to just copy the required binaries
+FROM debian:bullseye as build
+
+COPY stage-binary-and-deps.sh /
+COPY get-binaries-from-package.sh /
+
+ARG STAGE_DIR="/opt/stage"
+RUN apt -y update && \
+    apt -y install bash && \
```

Comment:
Bash is necessary on the base image, as stage-binary-and-deps have specific bash directives (like pipefail, etc)

Reply:
can we use bash-static instead of bash?

Reply:
@dims this script + bash is only used at build-time in a multi-stage build, it won't be present in the final image either way so normal bash is probably fine.

Reply:
ah ok